GPU/GPUTracking/Global/GPUChainTracking.cxx

```diff
@@ -1017,15 +1018,14 @@ int GPUChainTracking::RunTPCClusterizer()
       }
 
       runKernel<GPUTPCCFDeconvolution, GPUTPCCFDeconvolution::countPeaks>(GetGrid(clusterer.mPmemory->counters.nDigits, ClustererThreadCount(), lane), {iSlice}, {});
-      if (DoDebugAndDump(RecoStep::TPCClusterFinding, 0, clusterer, &GPUTPCClusterFinder::DumpCountedPeaks, mDebugFile)) {
```

Comment:
In principal, if you want to call 2 dump functions, you can do it like here, this avoids the dual data copy if you call DoDebugAndDump twice. But doesn't matter for the debug code now, so I am merging this. Just a minor optimization for the future.